### PR TITLE
update to ACK runtime v0.20.1/sdk 1.44.93

### PR DESCRIPTION
**Pull request by jaypipes**
Signed-off-by: Jay Pipes <jaypipes@gmail.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/resource/queue/sdk.go [hunk 1]
```diff
@@ -223,6 +242,13 @@ func (rm *resourceManager) sdkUpdate(
 	latest *resource,
 	delta *ackcompare.Delta,
 ) (*resource, error) {
+	var err error
```
Comment (jljaco) [inline pkg/resource/queue/sdk.go:245]: Can we change the codegen such that this variable is declared in the return value part of the function block, one line above?

Comment (jaypipes) [reply pkg/resource/queue/sdk.go:245]: sure, I can do that in a future PR @jljaco

**Review by A-Hilaly — APPROVED** [review]
:+1: 
/lgtm

**Pull request merged**